bower.json

```diff
@@ -19,7 +19,8 @@
   "dependencies": {
     "purescript-halogen": "^5.0.0-rc.4",
     "purescript-routing": "^9.0.0",
-    "purescript-foreign-object": "^2.0.0"
+    "purescript-foreign-object": "^2.0.0",
+    "purescript-js-uri": "master"
   },
   "devDependencies": {
     "purescript-psci-support": "^4.0.0"
```

Comment:
Since you're supporting PureScript 0.14 now these should get a major version bump too.